en/peripherals/camera.md

```diff
@@ -114,25 +104,12 @@ PX4 emits the MAVLink [CAMERA_TRIGGER](https://mavlink.io/en/messages/common.htm
 If camera capture is configured, the timestamp from the camera capture driver is used, otherwise the triggering timestamp.
 :::
 
-Camera capture/feedback is enabled in PX4 by setting [CAM_CAP_FBACK = 1](../advanced_config/parameter_reference.md#CAM_CAP_FBACK).
-
-If mixers are used, then the capture pin used depends on the hardware:
-- Pixhawk FMUv5x boards use the board-specific camera capture pin (PI0).
-- Other board use FMU PWM pin 6 (hardcoded) for camera capture.
+## MAVLink Command Interface
```

Comment:
The driver accepts these MAVLink commands:
- [MAV_CMD_DO_TRIGGER_CONTROL](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_TRIGGER_CONTROL)
- [MAV_CMD_DO_DIGICAM_CONTROL](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_DIGICAM_CONTROL)
- [MAV_CMD_DO_SET_CAM_TRIGG_DIST](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_SET_CAM_TRIGG_DIST)

My understanding is that it ignores the other commands, such as those for the MAVLink Camera Protocol but simply forwards those through to a connected MAVLink camera right?

Reply:
I don't know the details here.

Reply:
Thanks. I've reviewed the spec and updated this section. I've also recommended using MAVLink cameras (i.e. highlighted the differences).

It might not be 100% accurate as it is untested but it does correctly identify the limits of each approach.